app/src/main/java/org/ometa/lovemonster/service/ResponseParser.java

```diff
@@ -196,6 +196,12 @@ private Calendar parseDatetime(@Nullable final String datetime) {
      *      the non-blank value, or null otherwise
      */
     private String parseString(final JSONObject jsonObject, final String attributeName) {
+        // JSONObject will helpfully coerce null to "null" for us.  We don't want that.
+        // http://code.google.com/p/android/issues/detail?id=13830
+        if (jsonObject.isNull(attributeName)) {
```

Comment:
I can't reproduce this in a unit test.

When I run the following:

```diff
diff --git a/app/src/test/java/org/ometa/lovemonster/service/ResponseParserTest.java b/app/src/test/java/org/ometa/lovemonster/service/ResponseParserTest.java
index d7e132d..2b7b9a0 100644
--- a/app/src/test/java/org/ometa/lovemonster/service/ResponseParserTest.java
+++ b/app/src/test/java/org/ometa/lovemonster/service/ResponseParserTest.java
@@ -85,9 +85,10 @@ public class ResponseParserTest {
         final List<Love> loves = responseParser.parseLoveList(Fixtures.getJsonObject("v1_loves-blank_fields.json"));

         assertNotNull("loves should not be null", loves);
-        assertEquals("should have 2 loves", 2, loves.size());
+        assertEquals("should have 3 loves", 3, loves.size());

         assertNull("empty string message should be set to null", loves.get(0).message);
         assertNull("blank string message should be set to null", loves.get(1).message);
+        assertNull("null string message should be set to null", loves.get(2).message);
     }
 }
\ No newline at end of file
diff --git a/app/src/test/resources/fixtures/v1_loves-blank_fields.json b/app/src/test/resources/fixtures/v1_loves-blank_fields.json
index fcc1704..bc79745 100644
--- a/app/src/test/resources/fixtures/v1_loves-blank_fields.json
+++ b/app/src/test/resources/fixtures/v1_loves-blank_fields.json
@@ -50,6 +50,23 @@
         "name": null,
         "username": "bar"
       }
+    },
+    {
+      "reason": "null message",
+      "message": null,
+      "private_message": true,
+      "id": 11642,
+      "created_at": "2015-11-18T15:55:46Z",
+      "user_to": {
+        "email": "foo@example.com",
+        "name": null,
+        "username": "foo"
+      },
+      "user_from": {
+        "email": "bar@example.com",
+        "name": null,
+        "username": "bar"
+      }
     }
   ],
   "pages": 444
```

it passes.  It would be nice to get a unit test failing for this bug.